.github/workflows/release-embedding-sdk.yml

```diff
@@ -37,8 +37,138 @@ jobs:
           triggered by: @${{ github.event.sender.login }}
           EOF
 
-  determine-version:
-    name: Determine SDK version
+  send-slack-notification-start:
+    name: "Send Slack notification: SDK release started"
+    needs: determine-sdk-version
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+    outputs:
+      slackMessageId: ${{ steps.send-slack-message-start.outputs.result }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: release
+
+      - name: Prepare build scripts
+        run: cd ${{ github.workspace }}/release && yarn && yarn build
+
+      - name: Cache build scripts
+        uses: actions/upload-artifact@v4
+        with:
+          name: build-scripts
+          path: release/dist/index.cjs
+
+      - name: Send build start message
+        id: send-slack-message-start
+        uses: actions/github-script@v7
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        with:
+          result-encoding: string
+          script: | # js
+            const { mentionUserByGithubLogin, sendSlackMessage, githubRunLink } = require('${{ github.workspace }}/release/dist/index.cjs');
+            const sdkVersion = "${{ needs.determine-sdk-version.outputs.sdk_version }}";
+
+            const title = `:rocket: *SDK release: ${sdkVersion}* :rocket:`
+            const runLink = githubRunLink(
+              `:building_construction: _CI Run_ :building_construction:`,
+              context.runId,
+              context.repo.owner,
+              context.repo.repo,
+            );
+            const body = `${runLink} - run by ${mentionUserByGithubLogin("${{ github.event.sender.login }}")}`
+            const sendSlackMessageResponse = await sendSlackMessage({
+              channelName: 'proj-embedding-sdk',
```

Comment:
All slack commands have the same channel name, btw, when testing I used `bot-testing.